recipes/bash_kernel/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "bash_kernel" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a08c84eddd8179de5234105821fd5cc210015671a0bd3cd0bc4f631c475e1670
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
should this be noarch? Again, I think this should have a direct dependency on python too. though maybe somebody at @conda-forge/jupyter can weigh in on the thoughts on directly depending on python vs implicitly doing so?

Reply:
Adding `python` is not a problem, but why should it be `noarch` when `bash` is not available on windows? I understand that this module could be installed under windows, and can be used if some version of bash is installed there though.

Reply:
Could it find bash with Windows Subsystem for Linux (WSL)?

You should just leave it as noarch if it passes the tests. noarch packages like this one are compild by linux only eventually, so for a strange package like this one, it might make sense to let it fail on windows in the staging area.

Reply:
Got it. Done.